oci/table_oci_ons_subscription.go

```diff
@@ -317,7 +312,7 @@ func getSubscriptionDeliveryPolicy(ctx context.Context, d *plugin.QueryData, h *
 		}
 	}
 
-	return nil, nil
+	return nil, err
```

Comment:
I still don't quite get if we'd ever hit this `return` statement, I think we may need it, but would we hit here if we couldn't find the subscription from the topic's list of subscriptions?

Reply:
Yes agree we will not hit this condition, but we need it.